ViewControllers/MostPopularCollectionViewController.swift

```diff
@@ -91,16 +82,34 @@ class MostPopularViewController: UIViewController, UICollectionViewDelegate {
         button.translatesAutoresizingMaskIntoConstraints = false
         button.setTitle("See more", for: .normal)
         button.titleLabel?.font = UIFont.systemFont(ofSize: 14, weight: .semibold)
-        button.tintColor = UIColor(named: "acidGreen")
+        button.tintColor = UIColor(named: "titleColor")
         button.alpha = 0
         return button
     }()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        title = "Most popular"
-        navigationController?.navigationBar.titleTextAttributes = [.foregroundColor: UIColor.white, .font: UIFont.systemFont(ofSize: 24, weight: .heavy)]
+        setupUI()
+        
+        movieDataService.getMostPopularMoviesList { [weak self] (moviesList: [Movie]?) in
+            if let moviesList = moviesList {
+                self?.movies = moviesList
+            }
```

Comment:
Having a movie list is a pre-requisite to the method execution so we should use a `guard` statement.
```suggestion
            guard 
                let self = self,
                let movieList = movieList else
            { return }
            self.movies = movieList
```